### DISCUSSION TITLE
Poll: 2023 DELPH-IN Summit dates

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#86] [Team Posts]
Dear DELPH-IN participants,

We plan to hold the 2023 DELPH-IN Summit as a hybrid event, with in person activities in A Coruña, Spain, either in the end of June or the first week of July 2023 (before ACL). 

Please respond to the following Doodle poll by Wednesday Oct 26 to indicate which days you would be able to attend (either in person or online):
https://doodle.com/meeting/participate/id/aAPyxnBe/vote

Thank you!
Emily, for the Standing Committee

**Comment by olzama** [comment]
Dear @delph-in/participants ! Last reminder to indicate your preferences for the week of 2023 DELPH-IN summit which will take place in A Coruña, Spain :). Please do so by Friday!

https://doodle.com/meeting/participate/id/aAPyxnBe/vote
